test.py

```diff
@@ -1293,7 +1293,7 @@ async def main() -> int:
 
     options = parse_cmd_line()
 
-    open_log(options.tmpdir, options.log_level)
+    open_log(options.tmpdir, f"test.py.{','.join(options.modes)}.log", options.log_level)
```

Comment:
So let's use dashes instead of commas, even if it doesn't matter for shell, it may look more nice.